terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/validation/ValidationTest.java

```diff
@@ -0,0 +1,283 @@
+/*
+ * Copyright (C) 2013-2015 terasoluna.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+package org.terasoluna.gfw.functionaltest.app.validation;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.terasoluna.gfw.functionaltest.app.FunctionTestSupport;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(locations = { "classpath:META-INF/spring/seleniumContext.xml" })
+public class ValidationTest extends FunctionTestSupport {
+
+    private static final String SUCCESS = "Validation successfully!";
+
+    public ValidationTest() {
+    }
+
+    @Test
+    public void test01_01_ByteMin_valid() {
+        click(By.linkText("@ByteMin Test"));
+
+        setText(By.id("userName"), "あいう");
+        click(By.id("btn_validate"));
+        assertThat(getText(By.id("message")), is(SUCCESS));
+    }
+
+    @Test
+    public void test01_02_ByteMin_invalid() {
+        click(By.linkText("@ByteMin Test"));
+
+        setText(By.id("userName"), "あいu");
+        click(By.id("btn_validate"));
+        assertThat(getText(By.id("userName.errors")),
+                is("must be grater than or equal 6 Bytes"));

```

Comment:
@ikeyat 
I corrected these messages and similar errors.
* `ByteMin` :: `must be greater than or equal to 6 Bytes`
* `ByteMax` :: `must be less than or equal to 6 Bytes`

And I corrected also `@Compare.Operator.GRATER*`, and related errors.
(it will pass the travis-ci after merge https://github.com/terasolunaorg/terasoluna-gfw/pull/431)

Please show commit log 305319170b7b7bdce9d6bdc4b733a436588eb424.